apps/linear_algebra/Makefile

```diff
@@ -7,8 +7,8 @@ EMIT_OPTIONS = stmt,assembly
 
 EIGEN_INCLUDES ?= -I/usr/include/eigen3
 CBLAS_LIBS ?= -lblas
-ATLAS_FLAGS ?= -DUSE_ATLAS
-ATLAS_LIBS ?= -L/usr/lib/atlas-base -lcblas -latlas
+ATLAS_FLAGS ?= -DUSE_ATLAS -I/opt/ATLAS/include
+ATLAS_LIBS ?= -L/opt/ATLAS/lib -lptcblas -latlas
```

Comment:
Oh, ok. Maybe add a comment in the makefile explaining that and describing
where to go to get the sources then.

On Tue, Feb 10, 2015 at 3:00 PM, Kree Cole-McLaughlin <
notifications@github.com> wrote:

> In apps/linear_algebra/Makefile
> <https://github.com/halide/Halide/pull/653#discussion_r24459754>:
>
> > @@ -7,8 +7,8 @@ EMIT_OPTIONS = stmt,assembly
> >
> >  EIGEN_INCLUDES ?= -I/usr/include/eigen3
> >  CBLAS_LIBS ?= -lblas
> > -ATLAS_FLAGS ?= -DUSE_ATLAS
> > -ATLAS_LIBS ?= -L/usr/lib/atlas-base -lcblas -latlas
> > +ATLAS_FLAGS ?= -DUSE_ATLAS -I/opt/ATLAS/include
> > +ATLAS_LIBS ?= -L/opt/ATLAS/lib -lptcblas -latlas
>
> Yeah, I was a bit unsure about checking these lines in. But installing it
> locally is not just to get the pthreads version, Rather the packaged
> versions are kind of useless, as the whole point of the library is that is
> tunes its performance for the specific machine during the build.
>
> I can revert these lines of the Makefile for this PR though.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/halide/Halide/pull/653/files#r24459754>.
>